docs/integrations/search_service.md

```diff
@@ -9,7 +9,7 @@ and 'search-content-deleted' (deleted) events, aligning with the `StandardPayloa
 - [Search metadata contract](https://github.com/ONSdigital/dis-search-upstream-stub/blob/main/docs/contract/resource_metadata.yml)
 - [Search upstream service endpoints spec](https://github.com/ONSdigital/dis-search-upstream-stub/blob/main/specification.yml)
 
-The CMS also provides a paginated Resource API endpoint with all published pages at `/v1/resources/` in the public facing instance. This is
+The CMS also provides a paginated Resource API endpoint with all published pages at `/v1/resources/`. This is
 used by the search service for reindexing.
```

Comment:
Question: If it's only used by Search, should it be enabled in the external env? Feels like something which only makes sense on internal?